doc/source/examples/multiphysics/sloshing-in-rectangular-tank/sloshing-in-rectangular-tank.rst

```diff
@@ -0,0 +1,189 @@
+- ``examples/multiphysics/sloshing-in-rectangular-tank/sloshing_2000/sloshing-in-rectangular-tank_Re2000.prm``
+
+-----------------------------
+Description of the case
+-----------------------------
+
+Predicting the dynamics of free surface waves is essential for many industrial applications (e.g. transport of liquified natural gas). Yet, simulating their dynamics is difficult, especially for high values of the Reynolds number. Indeed, in this case, the amplitude of the waves dampen very slowly, which leads to an oscillatory wave problem which is highly sensitive to the time integration scheme and the coupling between the Volume of Fluid solver and the Navier-Stokes solver. 
+
+In this problem, we simulate the damping of a small amplitude wave in a cavity of dimension :math:`(-1,-1)x(1,0.1)` . The initial height of the wave :math:`\xi (x)` is given by:
+
+.. math::
+
+  \xi = 1+0.01*sin(\pi(x+0.5))
+
+Four values of the Reynolds number are investigated: 2, 20, 200 and 2000. 
+
+--------------
+Parameter file
+--------------
+
+The results for this problem are highly sensitive to the accuracy of the time-stepping scheme. For this reason, we use a 2nd order backward differentiation scheme with a variable time step. The ``adaptive time step scaling`` is set to 1.025 to ensure that the time-step does not rise too quickly during the oscillations of the wave.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+      set method           = bdf2
+      set time end         = 50
+      set time step        = 0.01
+      set adapt            = true
+      set max cfl          = 0.25
+      set output name      = sloshing-in-rectangular-tank_Re20
+      set output path      = ./output_Re20/
+      set output frequency = 1
+      set adaptative time step scaling = 1.025
+    end
+
+The ``multiphysics`` subsection is used to enable the VOF solver.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Multiphysics
+    #---------------------------------------------------
+    subsection multiphysics
+      set VOF  = true
+    end 
+    
+
+In the ``initial condition``, we define the initial height of the wave ensuring that the interface (the 0.5 isovalue) lies at the right height.
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Initial condition
+    #---------------------------------------------------
+    subsection initial conditions
+      set type = nodal
+      subsection uvwp
+        set Function expression = 0; 0; 0
+      end
+    
+      subsection VOF
+        set Function expression =  if (y<=(0.01*sin(3.1416*(x+0.5))), min(0.5-(y-0.01*sin(3.1416*(x+0.5)))/0.0025,1), max(0.5-(y-0.01*sin(3.1416*(x+0.5)))/0.0025,0))
```

Comment:
Why are you using this kind of diffused initial condition instead of the projection step?

Reply:
Because this ensures that the interface lies exactly, exactly at the right position. I have found that for this case it helps compared to using the strict 1/0 and then diffusing. I can try double checking though to see if it still plays a role.